packages/linux-driver-addons/dvb/depends/media_tree/package.mk

```diff
@@ -2,8 +2,8 @@
 # Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="media_tree"
-PKG_VERSION="2019-01-10-15d90a6ae98e"
-PKG_SHA256="cf170bfc0e78959c6b289db6f4cf7ec7b400dd4704f45909e1535a935702e8bc"
+PKG_VERSION="2017-08-20-3b6471c7becd"
```

Comment:
so this is downgrade?

Reply:
nope this is the latest version ... 
the package uses the author date and not the commit date, this is terrible meh but why should it be easy and reasonable

Reply:
WTF... :smile: